application/common/components/passwordStore/Ldap.php

```diff
@@ -288,11 +289,11 @@ protected function encodeForUnicodePwdField($password)
     }
 
     /**
-     * @param \Adldap\Models\Entry $user
+     * @param \LdapRecord\Models\OpenLDAP\Entry $user
      * @param string $password
      * @throws \common\components\passwordStore\PasswordReuseException
      */
-    protected function updatePassword($user, $password)
+    protected function updatePassword(Entry $user, string $password): void
```

Comment:
Why is the PHPDoc the long path, but the parameter type shorter? Please LDAP_Entry would be more clear as to what Entry really means.